reddit_gold/public/static/css/gold.less

```diff
@@ -760,12 +771,32 @@ section#postcard {
         }
     }
 
+    .selectors .button {
+        border-radius: 0;
+        border-right-width: 0;
+
+        &:first-child {
+            border-top-left-radius: 5px;
+            border-bottom-left-radius: 5px;
+        }
+
+        &:last-child {
+            border-top-right-radius: 5px;
+            border-bottom-right-radius: 5px;
+            border-right-width: 3px;
+        }
+    }
+
     .button-big {
-        font-size: 26px;
-        line-height: 74px;
-        height: 70px;
+        font-size: 18px;
+        line-height: 34px;
+        height: 50px;
         border-color: #ff8415;
```

Comment:
Would there be benefit to changing the color values here and elsewhere to variables? :no_bell: